src/Api/Pokemon/Support/CombatPointsCalculator.php

```diff
@@ -33,7 +33,7 @@ public static function getLevel($cpMuliplier)
      */
     public static function getMaxCp($attack, $defense, $stamina)
     {
-        $maxCpMultplier = self::$LEVEL_TO_CP_MULTIPLIER[40.0];
+        $maxCpMultplier = self::$LEVEL_TO_CP_MULTIPLIER[40];
```

Comment:
I believe 40 should be turned into '40'.

`$maxCpMultplier = self::$LEVEL_TO_CP_MULTIPLIER['40'];`